### PR TITLE
fix space in the configuration file 

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Fix the extra space in the configuration file
Force trim to avoid future issues

## Why is it important?

Fix a broken configuration

Backport is not required since the original PR was not backported -> https://github.com/elastic/e2e-testing/pull/2603

### CODE REVIEW

File: .ci/.e2e-tests-beats.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ SUITES:
     scenarios:
       - name: "Fleet"
         tags: "fleet_mode_agent"
-        platforms: ["centos8_arm64", "centos8_amd64", " debian_10_arm64", "debian_10_amd64", "sles15"]
+        platforms: ["centos8_arm64", "centos8_amd64", "debian_10_arm64", "debian_10_amd64", "sles15"]
```
Comment (mdelapenya) [inline .ci/.e2e-tests-beats.yaml:16]: 😱

File: .ci/Jenkinsfile [hunk 2]
```diff
@@ -447,7 +447,9 @@ def checkTestSuite(Map parallelTasks = [:], Map item = [:]) {
     }
     def pullRequestFilter = scenario.containsKey('pullRequestFilter') ? scenario.pullRequestFilter : ''
     def tags = scenario.tags
-      platformsValue.each { platform ->
+      platformsValue.each { rawPlatform ->
+        // platform is not space based, so let's ensure no extra spaces can cause misbehaviours.
+        def platform = rawPlatform.trim()
```
Comment (mdelapenya) [inline .ci/Jenkinsfile:453]: 💯

**Review by mdelapenya — APPROVED** [review]
Fixed, cool!

image

**Review by amannocci — APPROVED** [review]
LGTM

**Pull request merged**